react-troubleshooter.md

```diff
@@ -0,0 +1,57 @@
+# React Troubleshooter
+
+## Common errors
+
+### __invariant violation__: 
+
+## Testing
+
+
+### The Dom
+When testing React components, there are times where you will need to [render into the document](https://facebook.github.io/react/docs/test-utils.html#renderintodocument) however there is a very clear warning in the documenation that states you will need global `window`, `window.document` & `window.document.createElement`.
+
+Luckily, Ollie B has made us [react-tests-globals-setup](https://github.com/holidayextras/react-tests-globals-setup) which solves this problem & is available on npm (`npm install react-tests-globals-setup --save-dev`).
```

Comment:
I don't think we should call out individuals. Remember these docs are public. Also what is calling out doing? If it so we can approach the person that is called out, what happens if they are not around any more?

If we must call people out, can we at least link to their GitHub user. That is what most open source projects do.